examples/custom_object.py

```diff
@@ -46,7 +46,9 @@
 def main():
     config.load_kube_config()
 
-    api = client.CustomObjectsApi()
+    # passing prefered_content_type to choose the required type of patch
+    api_client = client.ApiClient(prefered_content_type="application/merge-patch+json")
+    api = client.CustomObjectsApi(api_client)
```

Comment:
What if someone wants to send different types of patch in their client? Is it possible to expose an argument in each API to specify the content type for each request?

Reply:
Sure, we can add it to each patch method but I thought that something like the default-content-type could help to write more consistent code. I've never mixed type of patch in my code yet.

Reply:
I'm okay with either way

Reply:
I've dropped `preferred_content _type` option - it didn't support situation where different type of methods have different valid content types.